src/removed.rs

```diff
@@ -0,0 +1,35 @@
+use std::collections::HashSet;
+use std::iter::FromIterator;
+
+use mailmap::Author;
+
+lazy_static::lazy_static! {
+    static ref REMOVED: HashSet<mailmap::Author> = HashSet::from_iter([
```

Comment:
nit: probably add a comment linking to the teams repo PR so this can be found again when someone inevitably asks what this is for in a couple years

Reply:
Oddly enough I intended to do this and totally forgot.